### ISSUE TITLE
[Laravel 8] Issue with Guzzle dependency

### ISSUE THREAD

**Issue by johanvanhelden** [laravel-notification-channels/pushover#44]
Hi there!

Laravel requires [Guzzle 7](https://github.com/laravel/laravel/blob/master/composer.json#L14), but this package requires [Guzzle 6](https://github.com/laravel-notification-channels/pushover/blob/master/composer.json#L16). This is preventing me from updating to Laravel 8:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for laravel-notification-channels/pushover ^2.1 -> satisfiable by laravel-notification-channels/pushover[2.1.0, 2.1.1, 2.1.2].
    - Can only install one of: guzzlehttp/guzzle[7.0.1, 6.5.x-dev].
    - Can only install one of: guzzlehttp/guzzle[7.0.x-dev, 6.5.x-dev].
    - Can only install one of: guzzlehttp/guzzle[7.1.x-dev, 6.5.x-dev].
    - Conclusion: install guzzlehttp/guzzle 6.5.x-dev
    - Installation request for guzzlehttp/guzzle ^7.0.1 -> satisfiable by guzzlehttp/guzzle[7.0.1, 7.0.x-dev, 7.1.x-dev].

```

I noticed the Laravel 8 PR has been merged without updating Guzzle. 

Might I ask why Guzzle is still on 6?

**Comment by casperboone** [comment]
Oh no! I hope to be able to take a look at it later this week! (in the meantime a PR that fixes it is also welcome ofc)

**Comment by casperboone** [comment]
Just released 3.0.0 with (actual) Laravel 8 support and a few new methods. Please let me know if there are still any issues :)

**Comment by johanvanhelden** [comment]
Thank you @casperboone !
